client/ayon_core/tools/publisher/control.py

```diff
@@ -2472,8 +2670,15 @@ def _publish_iterator(self):
         self.publish_progress = self.publish_max_progress
         yield MainThreadItem(self.stop_publish)
 
-    def _add_validation_error(self, result):
+    def _add_validation_error(self, result: dict) -> None:
+        if not result["is_blocking"] and self._ignore_non_blocking_errors:
+            return
```

Comment:
Add a log here:
```suggestion
        if not result["is_blocking"] and self._ignore_non_blocking_errors:
            self.log.info(f"Ignoring non-blocking error {result['plugin']}")
            return
```

Reply:
I recycled here the _`set_error_title_if_missing` method that I removed from `PublishValidationErrors` to use it for the log:
```
    @staticmethod
    def _set_error_title_if_missing(plugin, error):
        """Ensure that the validation report item has a title."""
        if not error.title:
            error.title = getattr(plugin, 'label',
                                  plugin.__name__)

    def _add_validation_error(self, result: dict) -> None:
        self._set_error_title_if_missing(result["plugin"], result["error"])
        if not result["is_blocking"] and self._ignore_non_blocking_errors:
            self.log.info(
                f"Ignoring non-blocking error: {result['error'].title}")
            return
```
The verification that is still made in the original code in `PublishValidationErrors` will be useless but in the log we'll have the same error title that we have in the Report page of the Publisher.